agents/kamevent.go

```diff
@@ -77,7 +77,7 @@ func NewKamEvent(kamEvData []byte, alias, adress string) (KamEvent, error) {
 	if err := json.Unmarshal(kamEvData, &kev); err != nil {
 		return nil, err
 	}
-	kev[utils.CGROriginHost] = utils.FirstNonEmpty(kev[utils.CGROriginHost], alias, adress)
+	kev[utils.OriginHost] = utils.FirstNonEmpty(kev[utils.OriginHost], alias, adress)
```

Comment:
Is there a reason for renaming cgr_originhost to OriginHost? The problem is that we are building the event in Kamailio and there we should have separate naming from standard pseudovariables.